docs/general/administration/backup-and-restore.md

```diff
@@ -0,0 +1,72 @@
+---
+uid: backup-and-restore
+title: Backup and Restore
+---
+
+# Backup and Restore
+
+This guide documents how to both back up and then restore the data in your Jellyfin instance.
+
+## Why Backing Up is Important
+
+Backups are important in general for all digital data. There are myriad possible bugs and issues that can arise and cause data loss. Within Jellyfin, as mentioned above, there is always a possibility of bugs showing up, especially when you upgrade major releases (for instance, from Jellyfin 10.9.x to Jellyfin 10.10.x).
+
+In addition, Jellyfin does not have a downgrade mechanism. This is very important to understand; once your Jellyfin instance has been started with a new version, any pending migrations are immediately applied, and your Jellyfin data will no longer work with the old version. The **only** way to restore your active instance back to the old version is to restore a backup. Backups are thus critical if you want to test Unstable versions, or before upgradiing to the latest version. In fact, when testing Unstable, it is best to back up much more regularly, perhaps using a cron job or similar automatic mechanism, to ensure that your data can be recovered if you hit a major issue.
+
+Backups may also come in handy if you hit a bug. For instance, if a bug in the new version causes corruption of some aspect of the Jellyfin data, restoring from a backup means you can quickly restore to good data and apply any fixes to avoid a repeat. Without a backup, you would be completely out of luck in this situation.
+
+Finally, administrative mistakes, normal operation issues (e.g. filling up a disk), or general bit rot can cause corruption or issues that require a backup to be restored. It's always better to have one than not.
+
+## Taking a Backup
+
+1. Stop the running Jellyfin server. This is extremely important, as otherwise the database will be locked and might not be recoverable when restoring. Note that this will interrupt any playback.
+
+   * Any platform: Within the Jellyfin Dashboard, click "Shutdown". This should cleanly stop the process on all platforms, but if not, try one of the following.
+   * Docker: `sudo docker stop jellyfin`
```

Comment:
```suggestion
   * Docker: `docker stop jellyfin`
```

Docker best practices have you add your user to the docker group, instead of running the docker command with sudo.
I'd expect people who haven't done this to be familiar enough with their setup to know to add it if needed.